### ISSUE TITLE
Default tamed health and damage multipliers

### ISSUE THREAD

**Issue by Matteo2000** [cadon/ARKStatsExtractor#192]
Hi all,

Firstly, I love ASB - I think it's a very clever tool.

I wanted to check that we're confident that the default tamed multipliers for health and damage are really 0.2 and 0.17? The default settings in the Ark Server Manager are 0.23 and 0.2! So obviously there's a clash.

The following website seems to agree with ASM rather than ASB:
http://steamcommunity.com/workshop/filedetails/discussion/764365165/276237094322927518/

Thanks.

**Comment by cadon** [comment]
Hi, iirc these numbers were valid around a year ago and changed to 0.2 and 0.17 around february 2016. Do you have issues with the extraction on an official server?

I just checked it with the devkit 253.8, they seem still to be valid.

**Comment by Matteo2000** [comment]
Thanks for the reply. I haven't tried it on an official server but it seems to be working with 0.2 and 0.17 when I use it on my unofficial server. 

I think you've probably got the values correct, it's just strange that Ark Server Manager (which is used by so many people) has different defaults!

Thanks,

Matt

**Comment by Matteo2000** [comment]
p.s. I've been using ASB to help me re-balance my game for PvE a small tribe on an unofficial server. Let me know if you're interested in seeing my spreadsheets!

**Comment by cadon** [comment]
Hi, you mean the set of multipliers you use for your server? Sure!

**Comment by Matteo2000** [comment]
[Ark balancing v9.xlsx](https://github.com/cadon/ARKStatsExtractor/files/818301/Ark.balancing.v9.xlsx)

This is where I've got to with balancing. I've set it so that tamed dinos get as powerful as wild dinosaurs once fully levelled and with armor. It's all designed for a small pve tribe on a private server but it's highly customisable.

After speaking to you about the multipliers I emailed the guys that code Ark Server Manager. They agreed that their default numbers were incorrect and you'll see in their latest patch they've changed their multipliers to match the ones in Ark Smart Breeding!

Thanks,

Matt
